core/events/events_block_create.js

```diff
@@ -103,7 +103,7 @@ BlockCreate.prototype.run = function(forward) {
       const id = this.ids[i];
       const block = workspace.getBlockById(id);
       if (block) {
-        block.dispose(false);
+        block.dispose(true);
```

Comment:
This will actually heal the stack rather than triggering the animations. Another `true` argument should be added to trigger the animation.